### ISSUE TITLE
NuGet package references no longer work in F# Interactive (.NET Framework only) in VS 16.10.0

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/fsharp#11622]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/NuGet-package-references-no-longer-work-/1433170)._

---
[severity:It's more difficult to complete my work] [regression] [worked-in:16.9.5]

To reproduce:

- Ensure Tools > Options > F# Tools > F# Interactive > Use .NET Core scripting is set to False.
- Open F# Interactive.
- Type `#r "nuget: Newtonsoft.Json";; `

I get the following error message:

```
  #r "nuget: Newtonsoft.Json";;
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^

stdin(1,1): error FS3216: Package manager key 'nuget' was not registered in [C:\Users\mpattison\AppData\Local\assembly\dl3\4H0BMNOV. VYQ\TN065Y4K. YRA\97a5ac2c\31a0ed1b_8351d701; c:\program files (x86)\microsoft visual studio\2019\professional\common7\ide\commonextensions\microsoft\fsharp\Tools\], []. Currently registered: 
```

This applies to `fsi.exe`, `fsiAnyCpu.exe` and the .NET Framework version of F# Interactive in the IDE.

`dotnet fsi`, which uses .NET Core (or 5.0) is unaffected.

A workaround is to run `fsi.exe --compilertool:"C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\Common7\IDE\CommonExtensions\Microsoft\FSharp`.

I'm running Windows 10 Enterprise N, version 1909.

---
#### Feedback Bot on 5/26/2021, 07:51 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by cartermp** [comment]
Note: if you use .NET Core scripting, this is not an issue

**Comment by zyzhu** [comment]
As .NET Core scripting does not support script debugging yet, this is high severity at least in my workflow that prevents me from upgrading VS to any newer version.

**Comment by gusty** [comment]
@cartermp NET Core scripting is not an option for everyone. I strongly suggest to change severity to high. This issue has really a very high impact.

**Comment by KevinRansom** [comment]
Goodness, how did this happen.  The nuget packagemanager extension is not deployed, we will try to get a fix out pronto

**Comment by cartermp** [comment]
Thanks, we talked about this and decided to bump up the priority. We will try to fix this ASAP and service it in the VS 16.11 release (16.11 is a servicing bugs-only release).

**Comment by KevinRansom** [comment]
I was wrong, it's not fixed.

**Comment by KevinRansom** [comment]
This hasn't flowed to a VS release branch yet.

**Comment by markpattison** [comment]
Hi @KevinRansom ,

Any idea whether this will be available in the next couple of weeks?  I'm doing an internal demo at my company and it would be a lot smoother if this fix were available.

(It was me who raised the [original ticket](https://developercommunity.visualstudio.com/t/NuGet-package-references-no-longer-work-/1433170) - serves me right for upgrading VS straight away 😂)

Thanks!

**Comment by cartermp** [comment]
@markpattison this is scheduled to be in the 16.11 release, but we don't know the exact date when that ships just yet. But the fix is in, so it should light up when 16.11 releases.

**Comment by brettfo** [comment]
We've merged this fix into the internal VS 16.11 branch just this morning.  As @cartermp mentioned, we don't have an exact ship date yet, but it will be there.

Thanks for finding and filing the issue.

**Comment by fwaris** [comment]
should have read this before upgrading vs :(
